index.html

```diff
@@ -47,6 +47,148 @@ <h2 class="secondary-text">I'am a Software Developer</h2>
     </div>
   </section>
   </div>
-</body>
 
+  <div class="works">
```

Comment:
Kindly use a `<section>` tag. This is the correct sematic tag. Also screen readers need to beable to read your page.

Also this should use `display:grid` to arrange the cards. as per the project requirements
<img width="1659" alt="Screenshot 2022-10-29 at 18 16 29" src="https://user-images.githubusercontent.com/48546992/198839380-d29aad85-e598-49f7-b6e6-7106dbd00533.png">